### PR TITLE
Android 5 crashing and Borderless Ripple Effect on Android

### LINKED ISSUE #1774: [Bug] TouchEffect on Image crashes on Android 5.1
### Description

1.2.0 broke TouchEffect on Android 5 again..
See #969 for more details. This seems to be the same issue.

Error message is still
```
Java.Lang.NoSuchMethodError: 'no non-static method "Landroid/widget/ImageView;.getForeground()Landroid/graphics/drawable/Drawable;"'
```
### Steps to Reproduce

1. Run this app on Android 5 emulator (or real device): https://github.com/maxkoshevoi/NureTimetable
2. Add a timetable
3. Click on setting icon on the right of the added timetable

<details>

image

</details>

### Stack Trace

VS doesn't show it for this exception

### Basic Information

- Version with issue: 1.2.0 - 1.3.1
- Last known good version: 1.2.0-pre2
- IDE: VS2022 17.1 Preview 1.1
- Platform Target Frameworks: 
  - Android: 5.1
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### PR DESCRIPTION
**Pull request by rotorgames**
### Description ###

Fixes #1774

This PR is included the fix preventing crashing on Android API 20 if the `NativeAnimation` is applied to `ImageButton`
Also, PR allows making the Ripple effect borderless. 
Tested on Android API 20 -> 29
Dec-03-2021 19-55-00

### API Changes ###
* Added `NativeAnimationBorderless` AttachedProperty to `TouchEffect`

### PR Checklist ###

- [ ] Has a linked Issue, and the Issue has been `approved`
- [ ] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard

- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

**Comment by rotorgames** [comment]
Any updates around this PR?

**Comment by pictos** [comment]
@rotorgames can you link the issue that this PR should fix?

File: samples/XCT.Sample/Pages/Effects/TouchEffectPage.xaml [hunk 1]
```diff
@@ -17,6 +17,17 @@
         <StackLayout Padding="{StaticResource ContentPadding}"
                      Spacing="10">
 
+            <StackLayout
+                Orientation="Horizontal">
+                <Label
+                    HorizontalOptions="FillAndExpand"
+                    FontSize="Title"
+                    TextColor="Black"
```
Comment (nichojo89) [inline samples/XCT.Sample/Pages/Effects/TouchEffectPage.xaml:25]: Default TextColor for Label is already black

Comment (rotorgames) [reply samples/XCT.Sample/Pages/Effects/TouchEffectPage.xaml:25]: @nichojo89 No, default TextColor is Color.Default

**Comment by rotorgames** [comment]
> @rotorgames can you link the issue that this PR should fix?

I didn't find any open issues about this. But I reproduced the issue around Android 5 crashing when I tested this borderless enhancement.

**Comment by pictos** [comment]
@rotorgames can you create an issue describing the bug and an attached sample project on it? That we can track it better and test the fix with your scenario. Thanks in advance

**Comment by rotorgames** [comment]
@pictos I found the opened issue for this PR: https://github.com/xamarin/XamarinCommunityToolkit/issues/1774

@maxkoshevoi @AndreiMisiukevich @VladislavAntonyuk Take a look on my solution please.

**Review by VladislavAntonyuk — APPROVED** [review]

**Comment by rotorgames** [comment]
@jfversluis Do you have any information about next release ETA?

**Comment by pictos** [comment]
@rotorgames we're working to release soon-sh (;

**Pull request merged**